### ISSUE TITLE
Bug: Icons as semantic elements not read by screen readers correctly

### ISSUE THREAD

**Issue by adam-mcgee** [FortAwesome/Font-Awesome#20699]
### Bug description

When using icons as semantic elements, auto accessibility isn't working for Windows Narrator and Mac VoiceOver.

I would expect the title attribute of the `<i>` element to be read by these screen readers, as described in [the documentation](https://docs.fontawesome.com/web/dig-deeper/accessibility#icons-used-as-semantic-elements). What happens is that the screen reader doesn't read the `title` attribute.

The accessible text is also not visible in the Accessibility Tree in Chrome Developer Tools.

I have included a workaround for this issue in the Codepen, however this is not ideal.

I am not sure what to choose for the "Serving" field.

### Reproducible test case

https://codepen.io/adam-mcgee/pen/GgJzOqO?editors=1100

### Screenshots

Codepen using Mac VoiceOver:

![Image](https://github.com/user-attachments/assets/25e7e7ea-afa5-4bd5-9c6a-33aa6a41ae0f)

Accessibility view on Chrome:

![Image](https://github.com/user-attachments/assets/38b44d36-f4fc-4801-ac42-4a7af0bcce4e)

Additional screenshots/video can be provided

### Font Awesome version

v6.7.2

### Serving

Other (as specified in the bug description)

### Implementation

CSS

### Browser and Operating System

MacOS 14.7.6:

- Google Chrome 137.0.7151.120
- Safari 18.5
- Firefox 140.0.1
- Edge 137.0.3296.93

Windows 11, 23H2:

- Firefox 140.0.0

### Web bug report checklist

- [x] I have included a test case because my odds go _way_ up that the team can fix this when I do
- [x] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

As per https://docs.fontawesome.com/web/dig-deeper/accessibility#icons-used-as-semantic-elements, auto-accessibility works for SVG + JS implementations

In the codepen and in the bug description I can see that this is using CSS implementation

Am I missing something?

**Comment by adam-mcgee** [comment]
That's ok. We will use the workaround as described in the issue description above, or the suggested manual accessibility. Do you mind if I ask why auto-accessibility is not supported for Web Fonts + CSS?

**Comment by sensibleworld** [comment]
@adam-mcgee Many apologies for never following up on this issue. The simple reason is that, with the SVG/JS implementation, we can inject additional information when rendering the icons. When using web fonts/css, we're limited to what's already included in our font files, and what you manually add to your code.

Also a quick note that we've adjusted how we handle accessibility in version 7: https://docs.fontawesome.com/web/dig-deeper/accessibility
